pkg/cloud/config/watcher.go

```diff
@@ -239,31 +237,17 @@ type MultiCloudWatcher struct {
 }
 
 func (mcw *MultiCloudWatcher) GetConfigs() []cloud.KeyedConfig {
-	var multiConfigPath string
 
-	if env.IsKubernetesEnabled() {
-		multiConfigPath = path.Join(env.GetConfigPathWithDefault("/var/configs"), cloudIntegrationSecretPath)
-	} else {
-		multiConfigPath = env.GetCloudCostConfigPath()
-	}
+	multiConfigPath := env.GetCloudCostConfigPath()
+
 	exists, err := fileutil.FileExists(multiConfigPath)
 	if err != nil {
 		log.Errorf("MultiCloudWatcher:  error checking file at '%s': %s", multiConfigPath, err.Error())
 	}
 
 	// If config does not exist implies that this configuration method was not used
 	if !exists {
-		// check the original location of secret mount
-		multiConfigPath = path.Join("/var", cloudIntegrationSecretPath)
-		exists, err = fileutil.FileExists(multiConfigPath)
-		if err != nil {
-			log.Errorf("MultiCloudWatcher:  error checking file at '%s': %s", multiConfigPath, err.Error())
-		}
-
-		// If config does not exist implies that this configuration method was not used
-		if !exists {
-			return nil
-		}
```

Comment:
This block of code will not be necessary with 2.0 and was never necessary for opencost users